### DISCUSSION TITLE
Can I install on TP-Link ARCHER-C7 v5?

### DISCUSSION THREAD

**Discussion by ujai** [Ysurac/openmptcprouter#2450] [Q&A] [answered]
I have TP-Link ARCHER-C7 v5, currently running on DD-WRT v3.0-r42514 std. Is it possible to install openmptcprouter?

**Comment by Ysurac** [comment, marked as answer]
This can be compiled but there is not enough flash, not enough memory and a slow CPU. So it's not a great idea.

**Reply by ujai** [reply]
Thank you
